src/opera_disp_tms/search.py

```diff
@@ -110,14 +32,7 @@ def from_umm(cls, umm: dict) -> 'Granule':
 
         attributes = umm['umm']['AdditionalAttributes']
         frame_id = int(next(att['Values'][0] for att in attributes if att['Name'] == 'FRAME_ID'))
-
-        # TODO: get orbit from umm metadata when it becomes available
-        if frame_id in CAL_ASC_FRAMES:
-            orbit_pass = 'ASCENDING'
-        elif frame_id in CAL_DES_FRAMES:
-            orbit_pass = 'DESCENDING'
-        else:
-            orbit_pass = 'UNKNOWN'
+        orbit_pass = next(att['Values'][0] for att in attributes if att['Name'] == 'ASCENDING_DESCENDING')
```

Comment:
@forrestfwilliams as a bridge to v0.9 data, we can fall back on querying the orbit_pass direction from the frame db

Reply:
Done